rasa/nlu/constants.py

```diff
@@ -1,42 +1,50 @@
-TEXT_ATTRIBUTE = "text"
+TEXT = "text"
```

Comment:
how come? they tell you exactly what behind it, it is `"text"`

Reply:
it's hard to use the usage inside the name, because they're used in different places as literally placeholder for the word `text`

Reply:
Yep, but that means that these constants have different meanings in different places. I think if a constant has different meanings / usages, it should be different names. Otherwise there is no advantage to hardcoding them.

Reply:
different usage, but the usage is linked through this constants, if you look where they're used, it makes a lot of sense to have them like that